maven-wrapper-distribution/src/resources/mvnw

```diff
@@ -247,6 +247,21 @@ fi
 # End of extension
 ##########################################################################################
 
+# If specified, validate the SHA-256 sum of the Maven wrapper jar file
+wrapperSha256Sum=""
+while IFS="=" read key value; do
+  case "$key" in (wrapperSha256Sum) wrapperSha256Sum=$value; break ;;
+  esac
+done < "$MAVEN_PROJECTBASEDIR/.mvn/wrapper/maven-wrapper.properties"
```

Comment:
Are you certain that this is POSIX compliant?

Reply:
I would think so, that's pretty basic. Also, the same logic is already applied in the script during download.

Reply:
I recommend to test this with a POSIX only shell.

Reply:
Ran it through shellcheck and none of my code is flagged there.